src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -603,27 +628,34 @@ public Document describeDataAccessRequestFieldsById(String id, List<String> fiel
         return darManage;
     }
 
+    public List<DataAccessRequestManage> getDraftDataAccessRequestManage(Integer userId) {
+        List<DataAccessRequest> accessList = userId == null
+                ? dataAccessRequestDAO.findAllDraftDataAccessRequests()
+                : dataAccessRequestDAO.findAllDraftsByUserId(userId);
+        return createAccessRequestManageV2(accessList);
+    }
+
+
     public File createApprovedDARDocument() throws IOException {
         List<Election> elections = electionDAO.findDataAccessClosedElectionsByFinalResult(true);
         File file = File.createTempFile("ApprovedDataAccessRequests.tsv", ".tsv");
         FileWriter darWriter = new FileWriter(file);
         dataAccessReportsParser.setApprovedDARHeader(darWriter);
         if (CollectionUtils.isNotEmpty(elections)) {
             for (Election election : elections) {
-                Document dar = getDataAccessRequestByReferenceIdAsDocument(election.getReferenceId());
                 DataAccessRequest dataAccessRequest = findByReferenceId(election.getReferenceId());
                 User user = userDAO.findUserById(dataAccessRequest.getUserId());
                 try {
-                    if (Objects.nonNull(dar) && Objects.nonNull(user)) {
-                        Integer datasetId = dataAccessRequest.getData().getDatasetIds().get(0);
-                        String consentId = dataSetDAO.getAssociatedConsentIdByDataSetId(datasetId);
-                        Consent consent = consentDAO.findConsentById(consentId);
+                    if (Objects.nonNull(dataAccessRequest) && Objects.nonNull(dataAccessRequest.getData()) && Objects.nonNull(user)) {
+                        Integer datasetId = !CollectionUtils.isEmpty(dataAccessRequest.getData().getDatasetIds()) ? dataAccessRequest.getData().getDatasetIds().get(0) : null;
+                        String consentId = Objects.nonNull(datasetId) ? dataSetDAO.getAssociatedConsentIdByDataSetId(datasetId) : null;
+                        Consent consent = Objects.nonNull(consentId) ? consentDAO.findConsentById(consentId) : null;
                         String profileName = user.getDisplayName();
                         if (Objects.isNull(user.getInstitutionId())) {
-                          logger.warn("No institution found for creator of this Data Access Request");
+                            logger.warn("No institution found for creator of this Data Access Request");
```

Comment:
Here too, I see this error repeated which doesn't tell me much. Please add the user id and the dar reference id so we can better track these down in the logs.